### ISSUE TITLE
[Suggestion] Hover strafing

### ISSUE THREAD

**Issue by Magnymbus** [tsunamayo/Starship-EVO#2820]
I found some instances when I wanted to strafe around a target when fighting in a Hover tank with fixed weapons. I would suggest putting it on a toggle using tab. I would also suggest that It controls like jetpack mode, in that you use the mouse for yaw. Pitch is debatable. Also, it would make parking much easier.

**Comment by nukeguard** [comment]
I agree strafing while in hover vehicle would be great addition

**Comment by tsunamayo** [comment]
Yes it make sense but I would have to rethink all the controls scheme for hovercrafts...

**Comment by Magnymbus** [comment]
Maybe you could make it shift+a/d? Or make it a toggle in the hovercraft controller block rather than on tab for every hovercraft?

**Comment by nukeguard** [comment]
could the control scheme for hover be same as ship, just without the vertical control, I really like to be able to steer with mouse.

**Comment by WILDCATreactor** [comment]
are q and e used in hovercraft control? they could be used for strafing
